WalletWasabi.Gui/Global.cs

```diff
@@ -128,6 +128,13 @@ public async Task DesperateDequeueAllCoinsAsync()
 
 		private bool Initialized { get; set; } = false;
 
+		public Task InitializeBitcoinStore(Network network)
+		{
+			BitcoinStore = new BitcoinStore();
+			var bstoreInitTask = BitcoinStore.InitializeAsync(Path.Combine(DataDir, "BitcoinStore"), network);
+			return bstoreInitTask;
```

Comment:
Return with await here, because we always do that. However I was long wondering if we could just return `Task`s like this. Maybe it would worth looking it up what it causes and if it's better, then do it like this from now on.

Reply:
The reason I did it this way was because currently we initialize the task [here](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Gui/Global.cs#L147), but do not call it with `await` until [here](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Gui/Global.cs#L229)

Reply:
I don't get your point.

Reply:
Anyhow, as I said, it's not our coding convention, but it may be a good opportunity to discuss if this should be our new coding convention. It seems neither of us know or looked up what the code returning a task without awaiting does.